src/textual/demo_thin_scrollbar.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import annotations
```

Comment:
This probably isn't the right place for this, as its not part of the public API.

Suspect we can omit it entirely.

Reply:
Definitely. I made the same comment in the PR itself: it's here to show the intended use, including the ability to replace on a specific instance. But I didn't plan on leaving it in the repo, absolutely.